src/memory/bitmap.zig

```diff
@@ -0,0 +1,61 @@
+const ft = @import("../ft/ft.zig");
+
+pub const Bit = enum(u1) {
+	Taken,
+	Free,
+};
+
+pub const BitMap = struct {
+	const Self = @This();
+	const nbits: usize = 8 * @sizeOf(usize);
+
+	nb_obj: usize = 0,
+	bits: []usize = undefined,
+
+	pub const Error = error{ OutOfBounds };
+
+	pub fn init(self: *Self, addr: [*]usize, nb_obj: usize) void {
+		var len = Self.compute_len(nb_obj);
+
+		self.* = BitMap{};
+		self.nb_obj = nb_obj;
+		self.bits = addr[0..len];
+		for (self.bits) |*b| b.* = 0;
+	}
```

Comment:
I think the "norm" in zig for the init function is to be a "static" function that return the object like in https://ziglang.org/documentation/master/std/src/std/Random.zig.html#L35

Reply:
Done 8a8dcb3